jetty-util/src/main/java/org/eclipse/jetty/util/Trie.java

```diff
@@ -26,7 +26,7 @@
  *
  * @param <V> the Trie entry type
  */
-public interface Trie<V>
+public interface Trie<V> extends Index<V>
```

Comment:
So do we still really need Trie interface?  just for the unused method variations?

I like Index as a better name, which can then be directly implemented by the Trie implementations.   We should not ever need an polymorphic Trie, only polymorphic Indexes

Reply:
No, the `Trie` interface can and should go away.